lib/core/configUpdater.cpp

```diff
@@ -166,28 +168,19 @@ void configUpdater::rest_callback(connectionInstance &con, nlohmann::json &json)
             con.send_error(msg, HTTP_RESPONSE::BAD_REQUEST);
             return;
         } else {
-            // ...and for correct types
-            if (it.value().type_name() !=
-                _init_values[uri].at(it.key()).type_name()) {
-                std::string msg = fmt::format(
-                            "configUpdater: Update to endpoint '{}' contained" \
-                            " value '{}' of type {} (expected type {}).",
-                            uri.c_str(), it.key().c_str(),
-                            it.value().type_name(),
-                            _init_values[uri].at(it.key()).type_name());
-
-                WARN(msg.c_str());
-                con.send_error(msg, HTTP_RESPONSE::BAD_REQUEST);
-                return;
-            }
-
-            if (it.value().type() == nlohmann::json::value_t::number_float &&
-                _init_values[uri].at(it.key()).type() != it.value().type()) {
-                std::string msg = fmt::format(
-                            "configUpdater: Update to endpoint '{}' contained" \
-                            " value '{}' of type float (expected type {}).",
-                            uri.c_str(), it.key().c_str(),
-                            _init_values[uri].at(it.key()).type_name());
+            // ...and for correct types:
+            // If type_name differs, they are of different type and we care.
```

Comment:
It might be worth saying what we are trying to achieve here in more detail.  i.e. something like "Reject changes in general type (string vs. number vs. option), and reject changes between (un)signed integers and floats, but allow both signed and unsigned integers to be interchanged"   As written it's fairly hard to parse from the comment what this does.

Reply:
Done